source/docs/software/basic-programming/joystick.rst

```diff
@@ -121,7 +101,7 @@ POV
    :alt: The angles used by the code of the POV/D-pad with 0 at the top and continuing clockwise.
 
 
-On joysticks, the POV is a directional hat that can select one of 8 different angles or read -1 for unpressed.  The XboxController D-pad works the same as a POV.  Be careful when using a POV with exact angle requirements as it is hard for the user to ensure they select exactly the angle desired.
+On joysticks, the POV is a directional hat that can select one of 8 different angles or read -1 for unpressed.  The XboxController/PS4Controller D-pad works the same as a POV.  Be careful when using a POV with exact angle requirements as it is hard for the user to ensure they select exactly the angle desired.
 
 ``GenericHID`` Usage
 --------------------
```

Comment:
The example below uses 0 and 1 twice, but the note about zero and 1 applies to only the second usage